ImplementationGuide/markdown/Uebersicht.md

```diff
@@ -34,26 +34,20 @@ Ziel des _Smart App Launch_ ist es, ein Zugriffstoken von einem OAuth2-kompatibl
+- [SMART App Launch - EHR Launch](https://hl7.org/fhir/smart-app-launch/STU2.1/app-launch.html#step-2-launch-ehr):
 Ein Client kann aus dem Kontext eines anderen Systems (das dann in der Rolle des App-Launchers agiert) direkt innerhalb einer bestehenden _User Session_ gestartet werden, beispielsweise indem der eingeloggte Benutzer den Client startet und dieser durch das System in einer neuen Browserinstanz oder einem eingebetteten _iframe_ geöffnet wird.
 
-- [SMART App Launch - Standalone Launch](https://hl7.org/fhir/smart-app-launch/STU2/app-launch.html#launch-app-standalone-launch):
+- [SMART App Launch - Standalone Launch](https://hl7.org/fhir/smart-app-launch/STU2.1/app-launch.html#launch-app-standalone-launch):
 Clients, welche außerhalb eines anderen Systems gestartet werden (z.B. Mobile Apps welche Daten von einem ISiK-Ressourcenserver abfragen möchten), müssen ihren Kontext explizit über eine Nutzerinteraktion gegen eine als ISiK-App-Launcher fungierende Komponente aufbauen. Dieses kann beispielsweise ein Widget zur Auswahl des zu aktivierenden Patientenkontextes sein.
 
-Beide Launch-Sequenzen bedingen eine weitreichende Integration von Autorisierungsserver und App-Laucher, die gemeinsam den Launch-Kontext des Clients bestimmen und verwalten. Diese enge Bindung steht im Widerspruch zu den formulierten [Vorgaben für ISiK-Sicherheit in ISiK Stufe 4](Motivation.md) und erschwert die Nutzung von marktgängigen Produkten oder Open-Source-Bibliotheken zum Aufbau eines IAM-Subsystems. **Daher werden in ISiK Stufe 3 im Modul ISiK-Sicherheit zunächst nur Interaktionen gegen Systeme in der Rolle eines Ressourcenservers definiert, der ISiK-relevante FHIR RESTful APIs anbietet. Diese Ressourcenserver sind in ISiK Stufe 3 für ISiK-Sicherheit bestätigungsrelevant und werden im Folgenden als 'ISiK-Ressourcenserver' bezeichnet.** Die Vorgaben umfassen dabei insbesondere die SMART-Konfiguration (Schritt 3) sowie die Syntax und Semantik der über das Zugriffstoken vermittelten Autorisierungen (Schritt 6). 
+Beide Launch-Sequenzen bedingen eine weitreichende Integration von Autorisierungsserver und App-Laucher, die gemeinsam den Launch-Kontext des Clients bestimmen und verwalten. Dies Verknüpfung der beiden Komponenten ist der Spezifikation inhärent. Im Sinne der originalen SMART-on-FHIR-Spezifikation verhalten sich Autorisierungsserver und App-Launcher als eine Einheit. Entsprechend werde diese zusammen im Bestätigungsverfahren zertifiziert. Es sei darauf hingewiesen, dass dies explizit nur für diese Kombination gilt. Bestehende IAM-Subsysteme, für die Verwaltung von Benutzeridentitäten und Zugriffsrechte, sind hiervon nicht betroffen. Benutzeridentitäten können über OpenID Connect in den Autorisierungsfluss eingebunden werden. Die Entscheidung über die Zugriffsrechte entscheidetet ein Policy-Decision-Point in Kombination mit dem Ressourcenserver.
```

Comment:
issue: wenn App-Launcher auch zertifiziert werden soll, dann soll dies im Bereich "ISIK-Konformität" (Conformance.md) der Spec beschrieben werden (zusätzlich zu allen anderen bestätigungsrelevanten Rollen)

Reply:
question: was genau ist mit "weitreichende Integration von Autorisierungsserver und App-Laucher" sowie "Verknüpfung der beiden Komponenten" gemeint? warum ist diese notwendig (vllt. mit Beispiel)?

typo: in dem Abschnitt gibt es einige Typos bzw. Grammatikfehler